app/forms/provider_interface/offer_wizard.rb

```diff
@@ -51,10 +66,22 @@ def course_option
 
     def next_step(step = current_step)
       index = STEPS[decision.to_sym].index(step.to_sym)
+
+      if FeatureFlag.inactive?(:provider_ske) && decision.to_sym == :make_offer && index.zero?
+        # Jump SKE flow if feature is disabled
+        # get the index for the conditions page (which is after the ske flow)
+        index = STEPS[decision.to_sym].index(:ske_length)
+      end
       return unless index
 
       next_step = STEPS[decision.to_sym][index + 1]
 
+      if current_step.to_sym == :ske_standard_flow && no_ske_required?
+        # Jump the ske flow
+        index = STEPS[decision.to_sym].index(:conditions)
+        next_step = STEPS[decision.to_sym][index]
+      end
```

Comment:
This section might benefit from a second pass, but that can be done when the feature flag is removed.